### PR TITLE
[MODINV-1051] Disallow updating holdings ownership with related boundwith

### PR DESCRIPTION
**Pull request by RomanChernetskyi**
### Purpose
Disallow updating holdings ownership with related boundwith

### Approach
Check for boundWiths relations with incoming holdingsRecordIds, send error message for such holdings
Jira: https://folio-org.atlassian.net/browse/MODINV-1051

### CODE REVIEW

**Review by dmytrokrutii — APPROVED** [review]
Please update NEWS.md

**Review by KaterynaSenchenko — APPROVED** [review]

**Review by RuslanLavrov — APPROVED** [review]

File: src/main/java/org/folio/inventory/resources/UpdateOwnershipApi.java [hunk 1]
```diff
@@ -291,6 +295,39 @@ private CompletableFuture<List<String>> updateOwnershipOfHoldingsRecords(Holding
     }
   }
 
+  private CompletableFuture<List<UpdateOwnershipHoldingsRecordWrapper>> validateHoldingsRecordBoundWithParts(List<UpdateOwnershipHoldingsRecordWrapper> holdingsRecordWrappers,
```
Comment (RuslanLavrov) [inline src/main/java/org/folio/inventory/resources/UpdateOwnershipApi.java:298]: please add inventory-storage.bound-with-parts.collection.get permission for the /holdings/update-ownership endpoint

Comment (RomanChernetskyi) [reply src/main/java/org/folio/inventory/resources/UpdateOwnershipApi.java:298]: Done, thank you!

Comment (RuslanLavrov) [reply src/main/java/org/folio/inventory/resources/UpdateOwnershipApi.java:298]: thank you!

**Pull request merged**